examples/types.rs

```diff
@@ -1,37 +0,0 @@
-#![cfg_attr(feature = "strict", deny(warnings))]
-#![feature(target_feature)]
```

Comment:
What happened to `examples/types.rs` and `examples/wat.rs` ?

Reply:
Ah these just looked like things that were playing around and not actually legitimate examples (in the sense of things we wanted to maintain over time) so I just deleted them.

Reply:
I also thought this, but never really looked into them.